uktrade/great-cms
# Add subscribe to FTA updates form

This PR adds a new form in which users can subscribe to receive updates about selected free trade agreements.

The url for the form is hidden behind a feature flag `FEATURE_FTA_FORM`. This flag will be removed after release to production.

**How to test**
- Make sure you have directory-forms-api running and set the `GOV_NOTIFY_API_KEY` env var in the same repo to the value from the dev environment
- Set `FEATURE_FTA_FORM = True` in your great-cms env vars
- Go to `contact/free-trade-agreements
- Submit the form, you should receive an email to the address given

**Screenshots**
![Screenshot 2022-10-25 at 09 50 42](https://user-images.githubusercontent.com/22460823/197729737-6bb73a15-b7d1-4775-9a93-1015c4dacd90.png)

![Screenshot 2022-10-25 at 09 54 51](https://user-images.githubusercontent.com/22460823/197729863-b100b337-ad74-467d-abf4-71c0e540b278.png)


### Workflow


### Reviewing help


### Housekeeping

- [ ] Ran the `make manage download_geolocation_data` command

### Merging
